src/components/map/map.js

```diff
@@ -145,6 +146,8 @@ export default function Map({
         : stateGeoJson.features,
   });
 
+  const width = useWindowDimensions().width;
```

Comment:
nit, for future ref: `const { width } = useWindowDimensions();`

also my personal take on vars is that unless creating a name makes the code much clearer (like in the case of complex math, say), I typically only create new vars when it's used in more than one place, and otherwise just inline the fn where needed. Again not worth changing and just personal opinion